LayoutTests/svg/hittest/resources/svg-tooltip.js

```diff
@@ -11,7 +11,7 @@
 // Returns true if an element can return a title or not
 function canReturnTitle(element)
 {
-  if (!element || element.tagName == "svg" && element.ownerDocument instanceof SVGDocument)
+  if (!element || (element.tagName === "svg" && element.ownerDocument instanceof Document && element.namespaceURI === "http://www.w3.org/2000/svg"))
```

Comment:
```suggestion
  if (!element || (element.tagName === "svg" && element.ownerDocument instanceof Document && element.namespaceURI === "http://www.w3.org/2000/svg"))
  if (!element || (element.tagName === "svg" && !!element.parentElement))  
```

Reply:
Let me fix this above.

Reply:
@Ahmad-S792 This will pass.

Reply:
@karlcow - Thanks for your help, it works. Now we can land this.